src/test/javascript/unit/factories.spec.js

```diff
@@ -0,0 +1,19 @@
+describe('factories unit testing', function(){
+
+	var ideaFactories;
+
+	beforeEach(module('ideaFactories'));
+
+	beforeEach(inject(function(restFactory) {
```

Comment:
If that spec is going to be used for testing other factories too then tests for restFactory can be combined into sub spec inside that spec